src/main.rs

```diff
@@ -68,6 +68,10 @@ struct Options {
     #[structopt(value_name = "ARGS", last = true)]
     nvim_args: Vec<String>,
 
+    /// Window size
```

Comment:
These comments are used to present the description of the option when `gnvim --help` is run.

Assume the user can read this comment and provide a more descriptive message, e.g.:

`Geometry of the window in <width>x<height> format